site/pages/concepts/multi-step-cast-actions.mdx

```diff
@@ -0,0 +1,233 @@
+# Mutli-step Cast Actions
+
+Multi-step Cast Actions are similar to [Cast Actions](/concepts/cast-actions) with a difference that they return a frame, instead of showing a message. (see the [spec](https://warpcast.notion.site/Frames-Multi-step-actions-f469054de8fb4ffc8b8e2649a41b6ad9)).
```

Comment:
```suggestion
Multi-step Cast Actions are similar to [Cast Actions](/concepts/cast-actions) with a difference that they display a frame, instead of displaying a message. See the [spec](https://warpcast.notion.site/Frames-Multi-step-actions-f469054de8fb4ffc8b8e2649a41b6ad9).
```